instance/tasks.py

```diff
@@ -47,6 +50,9 @@
 
 logger = logging.getLogger(__name__)
 
+# Constants  ##################################################################
+
+KILL_ZOMBIES_CRON_SCHEDULE = os.environ.get("KILL_ZOMBIES_SCHEDULE", "0 0 */1 * *").split()
```

Comment:
@spokerman12, `0 0 */1 * *` is the same as the simpler `0 0 * * *`, no? Afaik, the `*/1` just means run on every day of the month which is what `*` also means.

Reply:
You are right regarding the schedule.
On the Jira task and all discussions it was always stated "environment variables" rather than Django settings.

Reply:
@spokerman12, you must've misunderstood what I meant probably because you still don't have a lot of context about Ocim. Ocim's Django settings are indeed populated via environment variables and that is what I meant. See [here](https://github.com/open-craft/opencraft/blob/master/opencraft/settings.py#L44) for example. Afaik, there is (almost) nothing that reads the environment variables outside of the Django settings.

Reply:
All discussions in the task referred to "environment variables".
I'll apply the requirement change early tomorrow.

Reply:
Applied